test_files/declare.sickle.ts

```diff
@@ -1,5 +1,21 @@
-declare module DeclareTest {
-  export class Foo {
-    constructor();
+Error at test_files/declare.in.ts:10:28: IndexSignature not implemented in externs for interface
+====
+declare module DeclareTestModule {
+  module inner {
```

Comment:
https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/Namespaces%20and%20Modules.md 

namespace inner is preferred